src/plugins/data-card-tool/components/case-attribute.tsx

```diff
@@ -166,21 +186,23 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
 
   const valueClassNames = classNames(
     `value ${attrKey}`,
-    { "editing": editFacet === "value" }
+    { "editing": editFacet === "value" },
+    {"has-image": gImageMap.isImageUrl(valueStr)}
+  );
+
+  const valueInputClassNames = classNames(
+    `value-input ${attrKey}`,
   );
```

Comment:
You don't need `classNames` for this:
```suggestion
  const valueInputClassNames = `value-input ${attrKey}`;
```